packages/grid/src/styled/pane/StyledPaneSplitterButton.ts

```diff
@@ -0,0 +1,122 @@
+import { rgba } from 'polished';
+import { ISplitterButtonProps, Orientation } from '../../types';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { StyledPaneSplitter } from './StyledPaneSplitter';
+
+const COMPONENT_ID = 'pane.splitter_button';
+
+interface IStyledSplitterButtonProps extends ISplitterButtonProps {
+  orientation: Orientation;
+}
+
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor('background', 0, props.theme);
+  const hoverColor = getColor('primaryHue', 100, props.theme);
+  const focusRingColor = getColor('primaryHue', 300, props.theme);
+  const activeColor = getColor('primaryHue', 300, props.theme);
+  const boxShadow = props.theme.shadows.lg('4px', '8px', rgba(4, 68, 77, 0.15));
+  const focusRingShadow = props.theme.shadows.md(focusRingColor!);
+
+  return css`
+    box-shadow: ${boxShadow};
+    background-color: ${backgroundColor};
+
+    &:hover {
+      background-color: ${hoverColor};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${boxShadow}, ${focusRingShadow};
+    }
+
+    &:active {
+      background-color: ${activeColor};
+    }
+  `;
+};
+
+const sizeStyles = (props: IStyledSplitterButtonProps & ThemeProps<DefaultTheme>) => {
+  const { base } = props.theme.space;
+  const isHorizontal = props.orientation === 'top' || props.orientation === 'bottom';
+  const isVertical = props.orientation === 'start' || props.orientation === 'end';
+  const buttonDimension = base * 6;
+  const defaultPosition = `${-base * 2 - 1}px`;
+  let top;
+  let left;
+  let right;
+  let bottom;
+
+  if (isHorizontal) {
+    switch (props.placement) {
+      case 'start':
+        top = defaultPosition;
+        left = `${-base * 2}px`;
+        break;
+      case 'center':
+        top = defaultPosition;
+        left = `calc(50% - ${buttonDimension / 2}px)`;
+        break;
+      case 'end':
+        top = defaultPosition;
+        right = defaultPosition;
+        break;
+    }
+  }
+  if (isVertical) {
+    switch (props.placement) {
+      case 'start':
+        top = `${base * 6}px`;
+        left = `${-base * 2}px`;
+        break;
+      case 'center':
+        top = `calc(50% - ${buttonDimension / 2}px)`;
+        left = defaultPosition;
+        break;
+      case 'end':
+        bottom = `${base * 6}px`;
+        left = defaultPosition;
+        break;
+    }
+  }
+
+  return css`
+    top: ${top};
+    right: ${right};
+    bottom: ${bottom};
+    left: ${left};
+    width: ${buttonDimension}px;
+    min-width: ${buttonDimension}px;
+    height: ${buttonDimension}px;
+  `;
+};
+
+export const StyledPaneSplitterButton = styled(IconButton).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledSplitterButtonProps>`
+  position: absolute;
+  opacity: 0;
+
+  ${StyledPaneSplitter}:hover &,
+  &[data-garden-focus-visible] {
+    opacity: 1;
```

Comment:
Opacity will definitely need transition treatment. The UI is distractingly "flicker-y" with the current styling. I'd recommend a `0.25s ease-in-out` for consistency with other system elements (the splitter itself or the table overflow button, for example). You'll probably need to pull in existing `Button` transition info for the override.